core/capabilities.go

```diff
@@ -115,6 +116,7 @@ var CapabilityNameLookup = map[Capability]string{
 	Capability_ImageToVideo:               "Image to video",
 	Capability_Upscale:                    "Upscale",
 	Capability_AudioToText:                "Audio to text",
+	Capability_LLM:                        "Large Language Model",
```

Comment:
@kyriediculous can you please commit this so we can merge 🙏🏻.
```suggestion
	Capability_LLM:                        "Large language model",
```